### PR TITLE
MAT-5880: Reusable Integer Component

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-5880](https://jira.cms.gov/browse/MAT-5880)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/common/IntegerInput/IntegerInput.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import {
+  AutoComplete,
```
Comment (RohitKandimalla) [inline src/components/common/IntegerInput/IntegerInput.tsx]: AutoComplete? Do we need it ?

File: src/components/common/IntegerInput/IntegerInput.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
```
Comment (RohitKandimalla) [inline src/components/common/IntegerInput/IntegerInput.scss]: Whats happening here ?

File: src/components/common/IntegerInput/IntegerInput.tsx (hunk 1 above)
Comment (ethankaplan) [reply src/components/common/IntegerInput/IntegerInput.tsx]: Hmm we don't!

File: src/components/common/IntegerInput/IntegerInput.scss (hunk 2 above)
Comment (ethankaplan) [reply src/components/common/IntegerInput/IntegerInput.scss]: Removed, that was for when field was a number field.

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**